src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.IncrementalAnalyzerProcessor.cs

```diff
@@ -77,7 +77,7 @@ private partial class IncrementalAnalyzerProcessor
                     }
 
                     // event and worker queues
-                    _documentTracker = _registration.Workspace.Services.GetService<IDocumentTrackingService>();
+                    _documentTracker = _registration.Workspace.Services.GetRequiredService<IDocumentTrackingService>();
```

Comment:
I thought this was possible to be null in the OOP process.

Reply:
It is no longer possible to be null. A default implementation is provided in which no document is ever active.